Doc/c-api/object.rst

```diff
@@ -395,3 +395,42 @@ Object Protocol
    returns ``NULL`` if the object cannot be iterated.
 
    .. versionadded:: 3.10
+
+.. c:function:: void *PyObject_GetTypeData(PyObject *o, PyTypeObject *cls)
+
+   Get a pointer to subclass-specific data reserved for *cls*.
```

Comment:
For a new extension developer it may seem like getting class-specific metadata. Probably, something like this (but less clumsy) would be better:

```suggestion
   Get a pointer to a part of instance memory space of *o* reserved for *cls* level of an inheritance chain.
```

Reply:
Hm. Would this work?
```suggestion
   Get a pointer to the memory space of *o* reserved for data specific to subclass *cls*.
```

Reply:
> Hm. Would this work?

Looks good.